app/presenters/publishing_api/case_study_presenter.rb

```diff
@@ -54,7 +54,11 @@ def details
         first_public_at: first_public_at,
         format_display_type: item.display_type_key,
       }
-      details_hash[:image] = image_details if image_available?
+      if image_available? && image_required?
+        details_hash[:image] = image_details
+      else
+        details_hash[:image] = { url: "", caption: nil, alt_text: "", }
```

Comment:
Does this actually set the image as `null` in content store, or does it set it to this JSON?

If it's the latter, is that what we want? It seems an odd thing to set in the content store, though I don't know much about it.

Reply:
My mistake that commit message is very misleading! We do set it to this JSON because the content-schema requires url and alt_text to be a string, so we couldn't set it as nil. 

If we don't set the image at all then it picks up on the organisation image, this was our way of stopping it from grabbing any organisation image that exists